askama_parser/src/expr.rs

```diff
@@ -72,17 +75,74 @@ pub enum Expr<'a> {
 }
 
 impl<'a> Expr<'a> {
-    pub(super) fn arguments(i: &'a str, level: Level) -> ParseResult<'a, Vec<Self>> {
+    pub(super) fn arguments(
+        i: &'a str,
+        level: Level,
+        is_jinja_macro: bool,
```

Comment:
Prefer to rename this to `is_template_macro`, since this isn't Jinja.

Reply:
:+1: